backend/corpora/common/auth0_manager.py

```diff
@@ -47,20 +47,18 @@ def __getattr__(self, item):
     @staticmethod
     def get_auth0_management_token(domain: str) -> str:
         # Generate management token
-        payload = json.dumps(
-            dict(
+        payload = dict(
                 client_id=CorporaAuthConfig().client_id,
                 client_secret=CorporaAuthConfig().client_secret,
                 grant_type="client_credentials",
                 audience=f"https://{domain}/api/v2/",
             )
-        )
         headers = {
             "cache-control": "no-cache",
             "content-type": "application/json",
         }
 
-        response = requests.post(f"https://{domain}/oauth/token", data=payload, headers=headers)
+        response = requests.post(f"https://{domain}/oauth/token", json=payload, headers=headers)
```

Comment:
Should this be `data=payload` since it's a `dict` now?

Reply:
actually the opposite. The `json` parameter converts whatever you pass into JSON. `data` assumes you've already converted.

Reply:
This is a feature of the requests library. Look at the bottom of this section https://2.python-requests.org/en/master/user/quickstart/#more-complicated-post-requests.

Reply:
Ack, sorry, my mistake!